Firebase/Auth/Source/FIRAuth.m

```diff
@@ -281,8 +281,8 @@ + (FIRAuth *)auth {
     [NSException raise:NSInternalInconsistencyException
                 format:@"The default FIRApp instance must be configured before the default FIRAuth"
                        @"instance can be initialized. One way to ensure that is to call "
-                       @"`[FIRApp configure];` is called in "
-                       @"`application:didFinishLaunchingWithOptions:`."];
+                       @"`[FIRApp configure];` (`FirebaseApp.configure()` in Swift) in the App "
+                       @"Delegate's `application:didFinishLaunchingWithOptions:`."];
```

Comment:
Do we need to refer to the Swift method name for the app delegate method as well here as well?